client/stylesheets/_toCat.scss

```diff
@@ -1161,7 +1161,7 @@ HELP | ABOUT | FAQ | SAFETY | PRIVACY | CONTACT **/
 }
 
 .abouttext textarea {
-    width: 300% !important;
+    width: 95% !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered border, border-radius, height, padding, vertical-align, width